server/controllers/boardSettingsController.js

```diff
@@ -16,14 +15,17 @@ const getMembers = async function (req, res) {
     }
 
     const boardId = req.params.id;
-    const { members } = await Board.findOne({ id: boardId }, { members: 1 }).populate(
+    const board = await Board.findOne({ id: boardId }, { members: 1 }).populate(
       'members.user',
       'name email',
     );
+    if (!board) {
+      return res.status(400).send(buildResponse(false, 'Member does not exist'));
+    }
 
-    return res.send(members);
+    return res.send(buildResponse(true, '', board.members));
   } catch (exception) {
-    return res.status(500).send(buildResponse(false, `${exception}`));
+    return res.status(500).send(buildResponse(false, constants.SERVER_ERROR_MESSAGE));
```

Comment:
Maybe it's good if we can log the exception as it would be lost since we are sending custom message in response

Reply:
Did you mean console log

Reply:
Yes, console will work. Because if something goes wrong we will not get what exactly happened. We can later write middleware for it as well.